### PR TITLE
Feat/load bps

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### What does this PR do?

- Resolve bp load info

### Steps to test

1. run `yarn`
1. run `yarn start`
1. go to `localhost:3000`

### CODE REVIEW

File: src/routes/Home/FrontLayer/Home/Home.frontLayer.js [hunk 1]
```diff
@@ -16,9 +16,41 @@ const useStyles = makeStyles(styles)
...
   const { t } = useTranslation('homePage')
-  const [state] = useSharedState()
+  const [bpsData, setBpsData] = useState([])
+
+  useEffect(() => {
+    const loadBps = async () => {
+      const bpNames = await novotebuyeosUtil.getBps()
+      const bps = (
+        await Promise.resolve(
+          bpNames.reduce(async (previous, current) => {
```
Comment (leisterfrancisco) [inline src/routes/Home/FrontLayer/Home/Home.frontLayer.js:26]: @tetogomez please, let me know if there is a better way to handle a reduce function with an async call inside, or well, if you could realize a better approach to do the dame a reduce function does.

Comment (tetogomez) [reply src/routes/Home/FrontLayer/Home/Home.frontLayer.js:26]: nice!

**Review by tetogomez — APPROVED** [review]
LGTM

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**